guides/source/active_record_validations.md

```diff
@@ -81,7 +81,7 @@ validations.
 There are two kinds of Active Record objects - those that correspond to a row
 inside your database and those that do not. When you instantiate a new object,
 using the `new` method, the object does not get saved in the database as yet.
-Once you call `save` on that object then will it be saved into the appropriate
+Once you call `save` on that object, it will be saved into the appropriate
```

Comment:
```suggestion
Once you call `save` on that object, it will then be saved into the appropriate
```

Probably this, to have continuation from previous sentence